tools/build_third_party_libs.py

```diff
@@ -53,10 +47,23 @@
         cxx_flags = "'-std=c++20 -mmacosx-version-min=10.14'"
     elif sys.platform == "linux":
         platform_dir = "Linux"
-        cxx_flags = "'-std=c++20 -stdlib=libc++'"
+        cxx_flags = "-std=c++20 -stdlib=libc++"
```

Comment:
Are the quotation rules different on Linux? That seems weird. Also, if you don't want to deal with appending to `cxx_flags`, it wouldn't be the end of the world to just re-assign to it.

```
if whatever:
    cxx_flags = "-std=c++20 -stdlib=libc++"

...

if something_else:
    cxx_flags = "-std=c++20 -stdlib=libc++ -bunch -of -extra -stuff"
```